es/appendices/glossary.rst

```diff
@@ -4,41 +4,44 @@ Glossary
 .. glossary::
 
     routing array
-        An array of attributes that are passed to :php:meth:`Router::url()`.
-        They typically look like::
+        (Enrutamiento de array)
+        Un array de atributos que son pasados a :php:meth:`Router::url()`.
+        Usualmente se tipean de la siguiente manera::
 
             array('controller' => 'posts', 'action' => 'view', 5)
 
     HTML attributes
-        An array of key => values that are composed into HTML attributes. For example::
+    (Atributos HTML)
```

Comment:
you need more indentation here I think and some of the other below also. Your translation must not be aligned with the glossary element but with its detail